core/modules/system/system.menu.inc

```diff
@@ -130,7 +130,10 @@ function system_menu_block_form($config) {
     '#type' => 'checkbox',
     '#title' => t('Display menu toggle button on small screens'),
     '#default_value' => $config['toggle'],
-    '#description' => t('On devices with small screens, this option reduces the menu to a toggle button, commonly known as <a href="https://en.wikipedia.org/wiki/Hamburger_button" target="_blank"><em>hamburger button</em></a>. Clicking the icon reveals or hides the menu, saving screen space.'),
+    '#description' => t('On devices with small screens, this option reduces the menu to a toggle button, commonly known as <a href="https://en.wikipedia.org/wiki/Hamburger_button" target="_blank"><em>hamburger button</em></a>. Clicking the icon reveals or hides the menu, saving screen space. The breakpoint for small screens can be configured in the global !link_tag_openMenu settings!link_tag_close.', array(
+      '!link_tag_open' => user_access('administer menu') ? '<a href="/admin/structure/menu/settings" target="_blank">' : '',
+      '!link_tag_close' => user_access('administer menu') ? '</a>' : '',
+    )),
   );
```

Comment:
```suggestion
    '#description' => t('On devices with small screens, this option reduces the menu to a toggle button, commonly known as <a href="https://en.wikipedia.org/wiki/Hamburger_button" target="_blank"><em>hamburger button</em></a>. Clicking the icon reveals or hides the menu, saving screen space.'),
  );
  if (user_access('administer menu')) {
    $form['toggle']['#description'] .= t('The breakpoint for small screens can be configured in the global <a href="/admin/structure/menu/settings" target="_blank">Menu settings</a>.')
  }
```

Reply:
@klonos The problem with your suggestion is that users without the `administer menu` permission will never see this text (which might be helpful for them). The idea with my code was that they'd still see the text, it just wouldn't be linked to an 'Access denied' page if they didn't have permission.

Reply:
A string like `!link_tag_openMenu settings!link_tag_close` won't be able to be parsed by `potx` module, because it will think that `!link_tag_openMenu` is the placeholder.

I've seen the following approach which works well as long as the thing you're linking to has a string that can stand-alone easily for translators:
```
$link = user_access('administer_menu') ? l('admin/structure/menu/settings', t('Menu settings')) : t('Menu settings');
$description = t('The breakpoint for small screens can be configured in the global !menu_settings page.', array('!menu_settings' => $link));
```

Reply:
Yup, much better 👍

Reply:
Hehe, `l()` works the other way round - `l($text, $path, array $options = array())` (translated string first, path second). But I'm pretty sure, @BWPanda knows that anyway.